.github/workflows/format.yaml

```diff
@@ -0,0 +1,39 @@
+name: Format Codebase
+
+on:
+  push:
+    branches:
+    - master
+
+jobs:
+  format:
+    runs-on: ubuntu-latest
+
+    container: qmkfm/base_container
+
+    # protect against those who develop with their fork on master
+    if: github.repository == 'qmk/qmk_firmware'
+
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        fetch-depth: 1
+    - name: Install dependencies
+      run: |
+        apt install dos2unix
+        pip3 install -r requirements.txt
```

Comment:
Is this required when using the qmk docker image?

Reply:
If a PR added a new dependency, I would say its safest to run the install, rather than fail and wait for it to be bundled as part of the base image.

If the base container does already contain everything, you *should* just get a bunch of `Requirement already satisfied` and it only take a couple seconds.

Reply:
Gotcha :)